src/schema/nodes.ts

```diff
@@ -1,6 +1,10 @@
 import { NodeSpec } from "prosemirror-model"
+import { bulletList, listItem, orderedList } from "prosemirror-schema-list"
 import { nodes as proseNodes } from "prosemirror-schema-basic"
 
 export const nodes: NodeSpec = {
   ...proseNodes,
+  ordered_list: orderedList,
+  bullet_list: bulletList,
+  list_item: listItem,
```

Comment:
I think we'll also need to extend these with the `content` and `group` for each list type since the exported values of the schema-list doesn't do that out of the box like nodes do in the schema-basic - presumable because they expect that to be used with the addListToNodes() function.
ie:
```
ordered_list: {
        ....orderedList,
        content: "list_item+",
        group: "block"
        },
list_item: {
        ...listItem,
        content: "paragraph block*",
      }
      ```